docs/yul.rst

```diff
@@ -198,6 +198,7 @@ has to be specified after a colon:
 
 .. code-block:: yul
 
+    // This will not compile (not implemented yet)
```

Comment:
Knowing what is not implemented will make it easier to spot that the comment is outdated once we do finally implement it:
```suggestion
    // This will not compile (uint32 type not implemented yet)
```